### ISSUE TITLE
Add styles for the Grid headers with frozen columns.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-themes#630]
Add styles for the Grid headers with frozen columns. Now the styles are inline.

**Comment by dalingrin** [comment]
I have a dark theme and this required overriding a few styles with !important to workaround.
![image](https://user-images.githubusercontent.com/71467/53359048-7efb8080-38f7-11e9-9aae-bf3cda341438.png)

**Comment by PaitoAnderson** [comment]
Thanks for this, works great! Just pasting below to save others from retyping :)

```sass
/* kendo-react-grid v2.6.1 has hard coded background style */
.k-header,
.k-group-cell,
.k-grouping-row > td {
    background-color: $card-cap-bg !important;
}

/* Selected PanelBar color gets overriden by grid v2.6.1 workaround */
.k-link.k-header.k-state-selected {
    background-color: $accent !important;
}

/* tabstrip also had k-header class which is overriden by grid v2.6.1 workaround */
.k-tabstrip {
    background-color: transparent !important;
}
```

**Comment by joneff** [comment]
Fixed by #851, but the issue was not closed.
